spec/spec_helper.rb

```diff
@@ -149,6 +149,7 @@ module Shell
   config.filter_run_excluding :windows64_only => true unless windows64?
   config.filter_run_excluding :windows32_only => true unless windows32?
   config.filter_run_excluding :windows_nano_only => true unless windows_nano_server?
+  config.filter_run_excluding :windows_10_or_2016 => true unless windows_10_or_2016?
```

Comment:
For long term purposes, we probably mean 10 or newer? so would an more appropriate filter be something like `windows_gte_10` ?